README.md

```diff
@@ -56,16 +56,14 @@ In order to run the full suite of Acceptance tests, run `make testacc`.
 To run a specific acceptance test, run `go test -v ./sumologic  -run YourSpecificTestName`
 
 *Note:* 
-- Acceptance tests *create real resources*, and often cost money to run. The environment variables `SUMOLOGIC_ACCESSID`, `SUMOLOGIC_ACCESSKEY`, `SUMOLOGIC_ENVIRONMENT` / `SUMOLOGIC_BASE_URL`,  and `TF_ACC` must also be set for acceptance tests to work properly.
-  - For example, you can generate a personal access key in stag-alpha [here](https://cse-stag-alpha.stag.sumologic.net/ui/#/preferences). Once your test runs, you are then capable of viewing the real resources created by Terraform in the [stag-alpha UI](https://cse-stag-alpha.stag.sumologic.net/sec/hud?hours=24).
+- Acceptance tests *create real resources*, and often cost money to run. The environment variables `SUMOLOGIC_ACCESSID`, `SUMOLOGIC_ACCESSKEY`, `SUMOLOGIC_ENVIRONMENT` / `SUMOLOGIC_BASE_URL`,  and `TF_ACC` must also be set for acceptance tests to work properly. You can generate a personal access key from your environment. Once your test runs, you are then capable of viewing the real resources created by Terraform in the UI.
      ```sh
      $ export SUMOLOGIC_ACCESSID="yourAccessID"
      $ export SUMOLOGIC_ACCESSKEY="yourAccessKey"
-     $ export SUMOLOGIC_ENVIRONMENT="stag"
-     $ export SUMOLOGIC_BASE_URL="https://stag-api.sumologic.net/api/"
+     $ export SUMOLOGIC_ENVIRONMENT="yourEnvironment"
+     $ export SUMOLOGIC_BASE_URL="yourEnvironmentAPIUrl"
      $ export TF_ACC=1
      ```
-  - More information on configuration can be found at the [Terraform Provider codelabs documentation](https://github.com/Sanyaku/codelabs/blob/master/backend/pages/SumoLogicTerraformProvider.md).
```

Comment:
You could probably link to this doc for more info about these env vars
https://github.com/SumoLogic/terraform-provider-sumologic/blob/master/website/docs/index.html.markdown?plain=1#L110